src/lqcut.jl

```diff
@@ -88,97 +95,111 @@ function lq_drift_time_correction(
         t_lower = minimum(x_at_threshold)
         t_upper = maximum(x_at_threshold)
     else
-        error("Mode $mode not supported")
+        throw(ArgumentError("Drift time cutoff method $ctc_driftime_cutoff_method not supported"))
     end
 
     #store cutoff values in box to return later    
     box = (lq_lower = lq_lower, lq_upper = lq_upper, t_lower = t_lower, t_upper = t_upper)
```

Comment:
No need to change that here, but if you create a named tuple that has the field names identical to the variables that you're setting, you could write this in a more compact way:
```julia
box = (; lq_lower, lq_upper, t_lower, t_upper)
```